concepticondata/conceptlists/Carvalho-2021-206.tsv

```diff
@@ -0,0 +1,207 @@
+Carvalho-2021-206-132	132	moon	1313	MOON
+Carvalho-2021-206-133	133	night	1233	NIGHT
+Carvalho-2021-206-134	134	yesterday	1174	YESTERDAY
+Carvalho-2021-206-135	135	wind	960	WIND
+Carvalho-2021-206-136	136	rainbow	1733	RAINBOW
+Carvalho-2021-206-137	137	lake	624	LAKE
+Carvalho-2021-206-138	138	water	948	WATER
+Carvalho-2021-206-139	139	rain	658	RAIN (PRECIPITATION)
+Carvalho-2021-206-140	140	fire/firewood	2675	FIRE OR FIREWOOD
+Carvalho-2021-206-141	141	fire/firewood2	2675	FIRE OR FIREWOOD
+Carvalho-2021-206-142	142	hole	1667	HOLE
+Carvalho-2021-206-143	143	stone	857	STONE
+Carvalho-2021-206-144	144	pain	1783	PAIN
+Carvalho-2021-206-145	145	to be/have		
+Carvalho-2021-206-146	146	to arrive	1387	ARRIVE
+Carvalho-2021-206-147	147	to come	1446	COME
+Carvalho-2021-206-148	148	to return	581	COME BACK
+Carvalho-2021-206-149	149	to walk	1443	WALK
+Carvalho-2021-206-150	150	to pass	558	PASS BY
+Carvalho-2021-206-151	151	to step	2065	STEP (VERB)
+Carvalho-2021-206-152	152	to grow	479	GROW
+Carvalho-2021-206-153	153	to sing	1261	SING
+Carvalho-2021-206-154	154	to eat	1336	EAT
+Carvalho-2021-206-155	155	to swallow	897	SWALLOW
+Carvalho-2021-206-156	156	to say	1458	SAY
+Carvalho-2021-206-157	157	to speak	1623	SPEAK
+Carvalho-2021-206-158	158	to command	1998	COMMAND
+Carvalho-2021-206-159	159	to call	1084	CALL
+Carvalho-2021-206-160	160	to breathe	1407	BREATHE
+Carvalho-2021-206-161	161	to wash1	1453	WASH
+Carvalho-2021-206-162	162	to wash2	1453	WASH
+Carvalho-2021-206-163	163	to swim	1439	SWIM
+Carvalho-2021-206-164	164	to bath	138	BATHE
+Carvalho-2021-206-165	165	to plant/put	998	PUT
+Carvalho-2021-206-166	166	to dig	1418	DIG
+Carvalho-2021-206-167	167	to peel	275	PEEL
+Carvalho-2021-206-168	168	to hear	1408	HEAR
+Carvalho-2021-206-169	169	to see	1409	SEE
+Carvalho-2021-206-170	170	to learn	504	LEARN
+Carvalho-2021-206-171	171	to fear	1419	FEAR (BE AFRAID)
+Carvalho-2021-206-172	172	to sleep	1585	SLEEP
+Carvalho-2021-206-173	173	to lie down	215	LIE DOWN
+Carvalho-2021-206-174	174	to die	1494	DIE
+Carvalho-2021-206-175	175	to kill	1417	KILL
+Carvalho-2021-206-176	176	to cook	1100	COOK (SOMETHING)
+Carvalho-2021-206-177	177	to roast	1962	ROAST (SOMETHING)
+Carvalho-2021-206-178	178	to grind	1033	GRIND
+Carvalho-2021-206-179	179	to fetch	3551	FETCH
+Carvalho-2021-206-180	180	to sew	1457	SEW
+Carvalho-2021-206-181	181	pregnant	1123	PREGNANT
+Carvalho-2021-206-182	182	hungry	1533	BE HUNGRY
+Carvalho-2021-206-183	183	shy	487	SHY
+Carvalho-2021-206-184	184	hot	1286	HOT
+Carvalho-2021-206-185	185	cold	1287	COLD
+Carvalho-2021-206-186	186	wet	1726	WET
+Carvalho-2021-206-187	187	dry	1398	DRY
+Carvalho-2021-206-188	188	new	1231	NEW
+Carvalho-2021-206-189	189	heavy	1210	HEAVY
+Carvalho-2021-206-190	190	above	1741	ABOVE
+Carvalho-2021-206-191	191	below	1485	BELOW OR UNDER
+Carvalho-2021-206-192	192	core/middle	3208	CENTER OR MIDDLE
+Carvalho-2021-206-193	193	two	1498	TWO
+Carvalho-2021-206-194	194	three	492	THREE
+Carvalho-2021-206-195	195	cl.liquid		
```

Comment:
This is a classifier, so one could see if we have classifiers for liquids alrady, same cl.cylindrical below.

Reply:
There is DROP (CLASSIFIER) defined as 'Classifier for small units of liquid, like water'. I'm not sure it's broad enough here.